pkg/apis/flowcontrol/v1beta3/defaults.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta3
+
+import (
+	"k8s.io/api/flowcontrol/v1beta3"
+)
+
+// Default settings for flow-schema
+const (
+	FlowSchemaDefaultMatchingPrecedence int32 = 1000
+)
+
+// Default settings for priority-level-configuration
+const (
+	PriorityLevelConfigurationDefaultHandSize                 int32 = 8
+	PriorityLevelConfigurationDefaultQueues                   int32 = 64
+	PriorityLevelConfigurationDefaultQueueLengthLimit         int32 = 50
+	PriorityLevelConfigurationDefaultAssuredConcurrencyShares int32 = 30
+)
+
```

Comment:
the v1beta3 boundary is a chance to modify defaults from v1beta2 if needed... are there any changes to defaults that have been contemplated in the past that should be done at this point?

Reply:
We do not have any queued up.